src/components/auth-components/signin.js

```diff
@@ -118,6 +125,39 @@ const useStyles = makeStyles((theme) => ({
 export default function CenteredGrid() {
     const classes = useStyles();
 
+   // Using useState to store and update login info.
+
+    const [info, updateInfo] = React.useState({
+        username: '',
+        password: ''
+    });
+
+    // This function will be called upon clicking the button and will send the login info to the back end.
+
+    function sendInfo(){
+        axios.post(`http://localhost:5000/login`, querystring.stringify({username: info.username, password: info.password}), {
```

Comment:
Can we lose the localhost part here in this url?

Reply:
Will be resolved in latest commit!